client/my-sites/customer-home/main.jsx

```diff
@@ -196,6 +217,8 @@ const mapStateToProps = ( state ) => {
 			! isClassicEditor && 'page' === getSiteOption( state, siteId, 'show_on_front' ),
 		hasWooCommerceInstalled: !! ( installedWooCommercePlugin && installedWooCommercePlugin.active ),
 		isRequestingSitePlugins: isRequestingInstalledPlugins( state, siteId ),
+		isAtomicSiteLaunching:
+			isAtomic && ( getRequest( state, launchSite( siteId ) )?.isLoading ?? false ),
```

Comment:
Cool, TIL about `getRequest`, I hadn't thought about doing it that way.

Reply:
It was TIL for me also, stumbled on it in this [PR](https://github.com/Automattic/wp-calypso/pull/39527/files#diff-90952105113989ffcfa02e82bdbb4e4f305b509c51c2924c74ce27841c222d4d)